components/ide-proxy/components/ide-metrics-api/metrics.proto

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+syntax = "proto3";
+
+package ide_metrics_api;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/ide-metrics/api";
+option java_package = "io.gitpod.ide_metrics.api";
+
+service MetricsService {
+    rpc ReportMetric(ReportMetricsRequest) returns (ReportMetricsResponse) {
+        option (google.api.http) = {
+            post: "/reportMetric/{name}",
+            body : "*"
+        };
+    }
+    rpc reportLog(ReportMetricsRequest) returns (ReportMetricsResponse) {
```

Comment:
it should mimic https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events/report

How we are going to test in prev envs? cc @meysholdt